src/main/kotlin/uk/co/poolefoundries/baldinggate/input/EditorInputHandler.kt

```diff
@@ -0,0 +1,130 @@
+package uk.co.poolefoundries.baldinggate.input
+
+
+import com.badlogic.gdx.Input.Buttons
+import com.badlogic.gdx.Input.Keys
+import com.badlogic.gdx.InputProcessor
+
+interface EditorInputHandler {
+    fun pause()
+    fun leftClick(x: Int, y: Int)
+    fun rightClick(x: Int, y: Int)
+    fun dragCamera(deltaX: Float, deltaY: Float)
+    fun zoom(amount: Int)
+}
+
+object RawEditorInputHandler : InputProcessor {
+
+    private var lastX = 0f
+    private var lastY = 0f
+    private var middle = false
+    private var left = false
+    private var right = false
+    private val listener = LevelEditorInputProcessor
+    // todo get list of valid actions that aren't movement and display on UI
+
+    override fun keyDown(keycode: Int): Boolean {
+        return when (keycode) {
+//            Keys.ENTER -> {
+//                listener.endTurn()
+//                true
+//            }
+//            Keys.TAB -> {
+//                listener.inventory()
+//                listener.nextPlayer()
+//                true
+//            }
```

Comment:
You shouldn't leave commented out code in the codebase. You'll get told of for this in the industry.